src/components/config/configController.ts

```diff
@@ -0,0 +1,52 @@
+import ConfigRequest from "../../types/configRequest";
+import { Request, Response } from "express";
+import { Config } from "../../config";
+
+export const configController = (
+  req: Request<ConfigRequest>,
+  res: Response
+): void => {
+  const config = Config.getInstance();
+  if (req.body.clientId !== undefined) {
```

Comment:
There's probably something more generic we can do here by taking a list of allowed fields out of the body, then spreading them into the config

Reply:
https://github.com/alphagov/di-report-a-death/blob/main/src/common/forms/forms.ts#L79

Reply:
Not too fussed about this though

Reply:
Yeah, I debated over doing something like this, and decided for now it was probably alright, but I'm happy to be told otherwise